test/test_clever.py

```diff
@@ -22,46 +22,6 @@ def setUp(self):
       clever.api_base = os.environ.get('CLEVER_API_BASE', 'https://api.clever.com')
       if auth.get("token", None):
         clever.set_token(auth["token"])
-      elif auth.get("api_key", None):
-        clever.set_api_key(auth["api_key"])
-
-    def test_dns_failure(self):
```

Comment:
This was added since we wanted to ensure that DNS issues error out in a reasonable time.